bluetooth_mesh/messages/time.py

```diff
@@ -0,0 +1,150 @@
+#
+# python-bluetooth-mesh - Bluetooth Mesh for Python
+#
+# Copyright (C) 2019  SILVAIR sp. z o.o.
+#
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+#
+import enum
+
+from construct import (
+    Default,
+    ExprValidator,
+    GreedyRange,
+    Int8ul,
+    Int16ul,
+    Struct,
+    Switch,
+    obj_,
+    this, Int64ul, BitStruct, BitsInteger, Octet, Flag, Bytewise, Padding, BytesInteger, Padded, StopIf,
+)
+
+from .util import EnumAdapter, Opcode, EmbeddedBitStruct
+
+
+# fmt: off
+
+class TimeRoleEnum(enum.IntEnum):
```

Comment:
Drop `Enum` suffix

Reply:
I wanted to distinguish it from `TimeRole` parsing struct.

Reply:
Do it the other way around then - reusing `TimeRole` parsing struct is an implementation detail, while this one is visible to the user.